assessments/guide/tools.md

```diff
@@ -0,0 +1,11 @@
+# Tools
+
+This document contains a set of tools that will be useful to security reviewers and project leads.
+
+## Document conversion
+
+Self-assessments are sometimes done in a Google doc, which facilitates discussions. At the end of the
+review, the documents needs to be converted to markdown so that they can be added to the github repository.
+
+List of tools:
+- [Pandoc](https://pandoc.org/)
```

Comment:
```suggestion
- [Pandoc](https://pandoc.org/)
- [Google Docs extension: docs-to-markdown](https://chrome.google.com/webstore/detail/docs-to-markdown/igffnbdfnodiaphfmfaiiaegmoljbghf)
```

Reply:
I used [grip](https://github.com/joeyespo/grip), if I recall correctly, although I don't quite remember what I used for gdocs -> markdown (maybe some sed/awk or [docx2txt](https://github.com/ankushshah89/python-docx2txt) + manual edits)